Apps/WebClient/src/ClientApp/src/components/timeline/filters.vue

```diff
@@ -316,7 +322,12 @@ export default class FilterComponent extends Vue {
                                 {{ filter.display }}
                             </b-form-checkbox>
                         </b-col>
-                        <b-col cols="4" align-self="end" class="text-right">
+                        <b-col
+                            v-show="filter.isEnabled"
```

Comment:
Do we need this as if filter is not enabled, the whole row is hidden per line #222 above right?

Reply:
good catch! is updated now